### DISCUSSION TITLE
Add invert-scroll option

### DISCUSSION THREAD

**Discussion by parmort** [polybar/polybar#2803] [Ideas]
Add an `invert-scroll` boolean option for modules such as backlight. I have scrolling direction inverted on my arch install (as I dual-boot with Windows, which uses the opposite to arch defaults) and I find it a little not intuitive.

**Comment by patrick96** [comment]
You can basically already achieve that by setting `scroll-interval` to a negative number.

EDIT: The `scroll-interval` option is an unreleased feature though.
